src/client/conn/mod.rs

```diff
@@ -41,7 +41,8 @@
 //!
 //!     // To send via the same connection again, it may not work as it may not be ready,
 //!     // so we have to wait until the request_sender becomes ready.
-//!     request_sender.ready().await?;
+//!     // TODO:(mike) how to handle readiness
+//!     // request_sender.ready().await?;
```

Comment:
Oh, did I forget to add an `async fn ready` to the version-specific `SendRequest`s? That might be all that's needed, ya?

Reply:
Like this?

https://github.com/hyperium/hyper/pull/2961/commits/111295724536d767cb920d22dc6a9b8c68b17634